From 697d3abeff1563936da2653a8fa8db78d5dff671 Mon Sep 17 00:00:00 2001
From: antirez <antirez@gmail.com>
Date: Tue, 11 Apr 2017 16:56:00 +0200
Subject: [PATCH 07/19] Set lua-time-limit default value at safe place.

Otherwise, as it was, it will overwrite whatever the user set.

Close #3703.
---
 src/scripting.c | 1 -
 src/server.c    | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/scripting.c b/src/scripting.c
index 0a2ce1f3..216dfbf1 100644
--- a/src/scripting.c
+++ b/src/scripting.c
@@ -900,7 +900,6 @@ void scriptingInit(int setup) {
         server.lua_caller = NULL;
         server.lua_timedout = 0;
         server.lua_always_replicate_commands = 0; /* Only DEBUG can change it.*/
-        server.lua_time_limit = LUA_SCRIPT_TIME_LIMIT;
         ldbInit();
     }
 
diff --git a/src/server.c b/src/server.c
index 71bcda7d..609f3966 100644
--- a/src/server.c
+++ b/src/server.c
@@ -1533,6 +1533,7 @@ void initServerConfig(void) {
     server.migrate_cached_sockets = dictCreate(&migrateCacheDictType,NULL);
     server.next_client_id = 1; /* Client IDs, start from 1 .*/
     server.loading_process_events_interval_bytes = (1024*1024*2);
+    server.lua_time_limit = LUA_SCRIPT_TIME_LIMIT;
 
     server.lruclock = getLRUClock();
     resetServerSaveParams();
-- 
2.13.1

